runtime/syntax/man.vim

```diff
@@ -18,7 +18,7 @@ highlight default link manOptionDesc     Constant
 highlight default link manReference      PreProc
 highlight default link manSubHeading     Function
 
-if getline(1) =~# '^[^()[:space:]]\+([23].*'
+if b:sect =~# '^[23]'
```

Comment:
All of these buffer-local variables should still be prefixed with `man_`. We don't know what variables other plugins or user configs might be using.

```
b:man_sect
b:man_default_sections " looks like you updated this one in a later commit
```